src/components/Comments/Comments.module.css

```diff
@@ -24,146 +21,236 @@
-    padding-left: 81px;
-    padding-right: 94px;
+    padding: 32px 40px;
+    margin-top: 128px;
+    margin-bottom: 64px;
     background-color: var(--primary-background-color);
   }
+
   .commentsTitle {
-    font-family: var(--decoraction-font);
     color: var(--dark-text-color);
-    font-size: 49px;
+    font-size: 2.4375rem;
     margin-bottom: 64px;
   }
+
   .commentsContainer {
-    max-width: 1440px;
     position: relative;
-    padding-bottom: 64px;
-    padding-left: 15px;
-    gap: 40px;
-    flex-direction: row;
+    gap: 64px;
+  }
+
+  .leftSvg {
+    fill: var(--brand-text-color);
+    display: block;
+    position: absolute;
+    width: 58px;
+    height: 38px;
+    bottom: 164px;
+    left: 0;
   }
+
+  .rightSvg {
+    fill: var(--brand-text-color);
+    display: block;
+    transform: rotate(180deg);
+    position: absolute;
+    width: 58px;
+    height: 38px;
+    top: 0;
+    right: 0;
+  }
+
   .commentsBox {
     background-color: var(--secondary-background-color);
     border-radius: 0px 64px;
     display: flex;
+    height: 164px;
     justify-content: center;
     align-items: center;
     padding: 26px 62px;
   }
-  .commentsBox:nth-child(odd):before {
-    content: '“';
+
+  .commentsBox:nth-child(odd) {
+    margin-right: 238px;
+  }
+
+  .commentsBox:nth-child(even) {
+    margin-left: 238px;
+  }
+
+  .commentsText {
+    color: var(--light-text-color);
+  }
+
+  /* LANDING PAGE */
+
+  .commentsLandingPage {
+    padding: 32px 40px;
+    margin-top: 128px;
+    margin-bottom: 64px;
+  }
+  .commentsTitleLandingPage {
+    font-size: 2.4375rem;
+    margin-bottom: 64px;
+  }
+
+  .leftSvgLandingPage {
+    fill: var(--light-text-color);
+    display: block;
     position: absolute;
-    font-family: Lateef;
-    font-size: 200px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: normal;
-    opacity: 0.3;
-    color: var(--dark-brand-color);
-    top: 150px;
-    left: -6px;
-  }
-  .commentsBox:nth-child(even):after {
-    content: '“';
+    width: 58px;
+    height: 38px;
+    bottom: 164px;
+    left: 0;
+  }
+
+  .rightSvgLandingPage {
+    fill: var(--light-text-color);
+    display: block;
     transform: rotate(180deg);
     position: absolute;
-    font-family: Lateef;
-    font-size: 200px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: normal;
-    opacity: 0.3;
-    color: var(--dark-brand-color);
-    top: -198px;
-    right: -25px;
+    width: 58px;
+    height: 38px;
+    top: 0;
+    right: 0;
+  }
+
+  .commentsBoxLandingPage {
+    border-radius: 0px 64px;
+    display: flex;
+    height: 164px;
+    justify-content: center;
+    align-items: center;
+    padding: 26px 62px;
+  }
+
+  .commentsBoxLandingPage:nth-child(odd) {
+    margin-right: 238px;
+  }
+
+  .commentsBoxLandingPage:nth-child(even) {
+    margin-left: 238px;
+  }
+
+  .commentsTextLandingPage {
+    color: var(--dark-text-color);
+  }
+}
+
+@media screen and (min-width: 1440px) {
```

Comment:
I @media screen and (min-width: 1280px)  dla desktopów. Bo tak ogólnie to wszsytko wygląda ok. Kawał dobrej roboty zrobiłeś z stylami ;g